mailpoet/lib/Automation/Engine/Storage/WorkflowStorage.php

```diff
@@ -94,42 +105,48 @@ public function getWorkflows(array $status = null): array {
   }
 
   public function getWorkflowCount(): int {
-    $workflowTable = esc_sql($this->workflowTable);
-    return (int)$this->wpdb->get_var("SELECT COUNT(*) FROM $workflowTable");
+    $workflowsTable = esc_sql($this->workflowsTable);
+    return (int)$this->wpdb->get_var("SELECT COUNT(*) FROM $workflowsTable");
   }
 
   /** @return string[] */
   public function getActiveTriggerKeys(): array {
-    $workflowTable = esc_sql($this->workflowTable);
-    $versionTable = esc_sql($this->versionsTable);
-    $query = (string)$this->wpdb->prepare("
-      SELECT DISTINCT version.trigger_keys
-      FROM $workflowTable AS workflow, $versionTable as version
-      WHERE workflow.status = %s AND workflow.id=version.workflow_id
-      ORDER BY version.id DESC",
-        Workflow::STATUS_ACTIVE
-    );
-    $result = $this->wpdb->get_col($query);
+    $workflowsTable = esc_sql($this->workflowsTable);
+    $triggersTable = esc_sql($this->triggersTable);
 
-    $triggerKeys = [];
-    foreach ($result as $item) {
-      /** @var string[] $keys */
-      $keys = Json::decode($item);
-      $triggerKeys = array_merge($triggerKeys, $keys);
-    }
-    return array_unique($triggerKeys);
+    $query = (string)$this->wpdb->prepare(
+      "
+        SELECT DISTINCT triggers.trigger_key
```

Comment:
`triggers` as a table alias seems quite close to the `CREATE TRIGGER` command. It gets strangely highlighted by my PHPStorm. I didnt find any reserved word `triggers` in MySql, but maybe we should either escape or rename the alias, just to be on the safe side?

Reply:
👍 I'm also thinking about using short single-letter aliases. I think we're doing that in many parts of the MP codebase and it seems quite readable.